apps/deploy/ReadME.md

```diff
@@ -0,0 +1,123 @@
+<!--- Licensed to the Apache Software Foundation (ASF) under one -->
+<!--- or more contributor license agreements.  See the NOTICE file -->
+<!--- distributed with this work for additional information -->
+<!--- regarding copyright ownership.  The ASF licenses this file -->
+<!--- to you under the Apache License, Version 2.0 (the -->
+<!--- "License"); you may not use this file except in compliance -->
+<!--- with the License.  You may obtain a copy of the License at -->
+
+<!---   http://www.apache.org/licenses/LICENSE-2.0 -->
+
+<!--- Unless required by applicable law or agreed to in writing, -->
+<!--- software distributed under the License is distributed on an -->
+<!--- "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY -->
+<!--- KIND, either express or implied.  See the License for the -->
+<!--- specific language governing permissions and limitations -->
+<!--- under the License. -->
+
+
+How to Deploy TVM-VTA Modules
+=========================
+This folder contains an example on how to deploy TVM-VTA modules.
+It also contains an example code to deploy with C++ and Python.
+
+1. In host machine tvm project enable vta fsim or FPGA and compile tvm successfully.
+
+2. In target FPGA machine, flash bitstream into FPGA, following are example on pynq board
+
+   Run following command, "/home/xilinx/vta.bit" is the bitstream file
+
+	```bash
+        xilinx@pynq:~$ sudo python3
```

Comment:
it would be nice to have a simple python with a CLI that takes a file and downloads it. e.g. `python3 program.py --bit="/home/xilinx/vta.bit"`